src/sql/base/browser/ui/table/table.ts

```diff
@@ -121,6 +123,58 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 		this.mapMouseEvent(this._grid.onHeaderClick, this._onHeaderClick);
 		this.mapMouseEvent(this._grid.onDblClick, this._onDoubleClick);
 		this._grid.onColumnsResized.subscribe(() => this._onColumnResize.fire());
+
+		this._grid.onClick.subscribe((e, data) => {
+			if (this.isTreeGrid()) {
+				this.toggleTreeGridParent(data.row, data.cell);
+			}
+		});
+
+		this._grid.onKeyDown.subscribe((e, data) => {
+			if (this.isTreeGrid()) {
+				if ((<any>e).keyCode === 13) {
```

Comment:
please look for other places that we handle keyboard event and use the keycode name instead of number.